[DO NOT MERGE] Update buy link (#6)
microbit-matt-hillsdon/pxt-microbit: docs/courses/csintro/introduction.md

![micro:bit man](/static/courses/csintro/microbitman.jpg)

* micro:bits may be purchased from resellers:

> >   https://microbit.org/buy/ (you will need 1 micro:bit per student for this course).  The "micro:bit Go Kit" includes a battery pack and USB cable as well.

Author: ```suggestion
>   https://microbit.org/buy/ (you will need at least 1 micro:bit for this course).  The "BBC micro:bit Go" includes a battery pack and USB cable as well.
```
